### ISSUE TITLE
Page content spinner spins forever on org lookup error

### ISSUE THREAD

**Issue by KlaasH** [ImpactView/impact-view-philly#24]
I somehow ended up at http://localhost:9900/#/museum//.  I got there by trying to view details for something on the map, but I haven't managed to reproduce the situation by clicking things.  But obviously it's easy to reproduce by just typing that URL.

When you do, or when anything else goes wrong with the museum/org lookup, the spinner in the `main-content` section never stops spinning.

![image](https://user-images.githubusercontent.com/6598836/32461246-37365470-c303-11e7-9b41-401dc336447f.png)

**Comment by CloudNiner** [comment]
Yeah, it's subtle, but this happens when you click the "details" link in a popup when carto is slow and hasn't yet loaded the ID into the popup template. So the URL is `/#//`.
